man/systemd.network.xml

```diff
@@ -4630,6 +4630,15 @@ ServerAddress=192.168.0.1/24</programlisting>
           <para>Takes a boolean. Configures whether the port is "locked" and does not allow traffic forwarded
           until fully authenticated, e.g. via 802.1x. When unset, the kernel's default will be used.</para>
 
+          <xi:include href="version-info.xml" xpointer="v258"/>
+        </listitem>
+        <term><varname>MAB=</varname></term>
```

Comment:
Please do not use abbreviate word, so `MACAuthenticationBypass=`.

commit message: please use the full hash and mention that it is in v6.2 .